http-handler/src/main/java/com/flipkart/poseidon/handlers/http/impl/oauth/OAuthTokenResponse.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015 Flipkart Internet, pvt ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.flipkart.poseidon.handlers.http.impl.oauth;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.LowerCaseWithUnderscoresStrategy.class)
+public class OAuthTokenResponse {
```

Comment:
Can you confirm this is standard oauth token response and there is no customization here.

http://tutorials.jenkov.com/oauth2/authorization-code-request-response.html